src/t2iapi/device/types.proto

```diff
@@ -80,3 +82,26 @@ enum CalibrationState{
     CALIBRATION_STATE_OTHER = 4;
 }
 
+
+/*
+Contains the meta data of the device.
+ */
+message MetaDataMsg {
+    /*
+    Represents pm:Udi (defined in IEEE Std 11073-10207-2017).
+    */
+    message UdiMsg {
+      string device_identifier = 1;
+      string human_readable_form = 2;
+      t2iapi.biceps.InstanceIdentifierOneOfMsg issuer = 3;
+      t2iapi.biceps.InstanceIdentifierOneOfMsg jurisdiction = 4;
```

Comment:
I think we can simplify here and only use InstanceIdentifier, not the related OneOf. The fact that OperatingJurisdiction is allowed is more like a bug in BICEPS and we should not make this more complicated for our users than it needs to be.

Reply:
done